src/api/profile.ts

```diff
@@ -14,22 +14,18 @@ export interface MakeUpdateUserGridLocationProps {
 export const makeUpdateUserGridLocation = ({
   venueId,
   userUid,
-  profileData,
 }: MakeUpdateUserGridLocationProps) => (
```

Comment:
`MakeUpdateUserGridLocationProps` should also have been changed to remove this prop from it.